BrainPortal/app/views/signups/new.html.erb

```diff
@@ -24,180 +24,167 @@
+
+      <div class="tree-column-grid">
+
+        <%= f.label(:title, "Title:") %>
+        <%= f.text_field :title %> <span class="field_note">for example: 'Mrs.', 'Mr', 'Dr.', etc.</span>
+
+
+        <%= f.label(:first, "* First Name:") %>
+        <%= f.text_field :first %>
+        <br>
+
+        <%= f.label(:middle, "Middle Name:") %>
+        <%= f.text_field :middle %>
+        <br>
+
+        <%= f.label(:last, "* Last Name:") %>
+        <%= f.text_field :last %>
+        <br>
+
+        <%= f.label(:login, "Preferred 'login' name:") %>
+        <%= f.text_field :login %>
+
+
+
+        <span class="field_note">
+           one letter + alphanums. <strong>By convention</strong>: the first letter of your first name + last
+        </span>
+        <span></span>
+        <span></span>
+        <span>name. For examaple, John Doe login: 'jdoe'</span>
+
+      </div>
+
+    </fieldset>
+
+    <br>
+
+    <fieldset class="groupentry">
+
+      <legend>
+        Institution Info
+      </legend>
+
+      <div class="two-column-grid">
+        <%= f.label(:institution, "* Name of Institution/Organization:") %>
+        <%= f.text_field :institution, :size => 60 %>
+
+
+        <%= f.label(:department, "* Department") %>
+        <%= f.text_field :department, :size => 60 %>
+
+
+        <%= f.label(:position, "* Current position or role:") %>
+        <%= f.select :position, options_for_select(
+            [["(select one)", ""], "Faculty",
+                                   "Postdoctoral",
+                                   "PhD. candidate",
+                                   "Masters student",
+                                   "Student",
+                                   "Researcher",
+                                   "Other"
+            ],
+            @signup.position)
         %>
-     </td>
-    </tr>
-
-    <tr>
-      <td colspan="2">&nbsp;</td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:comment, "Comments or special requests:") %></th>
-      <td>
-          <%= f.text_area :comment, :cols => 80, :rows => 5 %>
-          <br>
-          <small>
-                 Please tell us the name of the laboratory you work for, the name of its Principal Investigator (if not you), and if possible anyone<br>
-                 else you know in your lab who are already CBRAIN users. We'll use this information to create or add you to a <em>Site</em> within CBRAIN.
+
+        <%= f.label(:affiliation, "* Current affiliation:") %>
+        <%= f.select :affiliation, options_for_select(
+            [["(select one)", ""], "Academic",
+                                   "Private sector",
+                                   "Government",
+                                   "Non-profit",
+                                   "Other",
+            ],
+            @signup.affiliation)
+        %>
+
+
+        <%= f.label(:email, "* Institutional Email address:") %>
+        <%= f.text_field :email, :size => 60 %>
+
+        <p></p>
```

Comment:
if you change the <p>s for <div>s instead, you will remove many of the extra vertical white space, so that the note about the email address is really just below the input text box